src/Microsoft.AspNetCore.Mvc.ApiExplorer/project.json

```diff
@@ -8,7 +8,9 @@
   "compilationOptions": {
     "warningsAsErrors": true,
     "keyFile": "../../tools/Key.snk",
-    "nowarn": [ "CS1591" ],
+    "nowarn": [
+      "CS1591"
+    ],
```

Comment:
Don't beleive so, this is newtonsoft reformatting when I spit the file back out.